articles/native-platforms/android/02-custom-login.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: Auth0 Android Quickstarts - Custom Login
+title: Custom Login
 description: This tutorial will show you how to use the Auth0 authentication API in your Android project to create a custom login screen.
 seo_alias: android
 ---
 
-## Android - Custom Login Tutorial
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/inaka/auth0-android-sample/tree/02-customlogin',
```

Comment:
To be updated to the valid URL once [this PR](https://github.com/auth0-samples/auth0-android-sample/pull/3) is reviewed and merged.